### DISCUSSION TITLE
Integrating CTI with security Onion

### DISCUSSION THREAD

**Discussion by ranimhassine** [Security-Onion-Solutions/securityonion#13475] [Unsupported Versions]
Hello i am currently using security onion 2.4.8
Can anyone provide any kind of guidance for the matter in integrating CTI with security onion

**Comment by InfosecGoon** [comment]
Are you using a threat intelligence platform like MISP or Anomali or are you just trying to write custom detections for discrete indicators?

**Reply by ranimhassine** [reply]
I am using ALIENVAULT OTX

**Reply by InfosecGoon** [reply]
If you want to download your indicators from OTX into Security Onion, you can do that with the appropriate Threat Intel Elastic Integration.

https://www.elastic.co/docs/current/integrations/ti_otx

Please note, this will just download the indicators and put them into your Elasticsearch database, it won't do any matching. If you want to automatically match indicators against your log data, you'll need to create an enrich pipeline. You may find this talk from last year's conference interesting:

https://www.youtube.com/watch?v=4TwstDJBiVw
